src/librustc_middle/mir/mod.rs

```diff
@@ -1861,7 +1861,7 @@ impl<'tcx> Copy for PlaceElem<'tcx> {}
 
 // At least on 64 bit systems, `PlaceElem` should not be larger than two pointers.
```

Comment:
This PR seems to make this comment out of date?

Reply:
I'm changing it, I haven't pushed it yet